### ISSUE TITLE
Build error: "no such file or directory, scandir"

### ISSUE THREAD

**Issue by mattdsteele** [ionic-team/stencil#578]
**Stencil version:**

```
 @stencil/core@0.6.11
```

**I'm submitting a:**

[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or https://stencil-worldwide.slack.com

**Current behavior:**
Trying to build from the component-starter template fails after "optimize html" step

**Steps to reproduce:**
1. Clone stencil-component-starter
2. Update to `@stencil/core@0.6.11`
3. Run `npm run build`

It should fail with this error:

```
C:\Users\IGEN261\code\github\err1>npm run build

> my-component@0.0.1 build C:\Users\IGEN261\code\github\err1
> stencil build

[43:15.5]  @stencil/core v0.6.11
[43:15.5]  build, mycomponent, prod mode, started ...
[43:15.5]  compile started ...
[43:17.9]  compile finished in 2.39 s
[43:18.1]  generate bundles started ...
[43:18.3]  generate bundles finished in 205 ms
[43:18.3]  generate app files started ...
[43:21.7]  generate app files finished in 3.40 s
[43:21.7]  optimize html, started: / ...
[43:22.9]  optimize html, finished: / in 1.18 s

[ ERROR ]  ENOENT: no such file or directory, scandir 'C:\Users\IGEN261\code\github\err1\dist\types' at Error (native)

[43:22.9]  build failed in 7.45 s
```

**Other information:**
Tested on both Linux and Win10. Node 6.12.0.

This started occuring with 0.6.9, which makes me think it might be an issue with the new Rollup?

**Comment by adamdbradley** [comment]
Should be fixed in 0.6.12 which was just released. Can you give it a test quick?

**Comment by mattdsteele** [comment]
Looks good, thanks!
